### ISSUE TITLE
autocomplete attributes ,WCAG 1.3.5 

### ISSUE THREAD

**Issue by asthaawasthi-laitkor** [dequelabs/axe-core#4281]
### Product

axe-core

### Question

does axe-core check for autocomplete attribute and relates with WCAG.

**Comment by straker** [comment]
Axe-core has the [autocomplete-valid](https://dequeuniversity.com/rules/axe/4.8/autocomplete-valid?application=RuleDescription) rule which checks that it's used with a valid value.

**Comment by laitkor** [comment]
Thanks @straker  However axe-core is not catching the input tag where the autocomplete attribute is missing

I have checked on this url www.villaserenaseniorapartments.com/gallery/

**Comment by straker** [comment]
@laitkor The autocomplete attribute is not required so axe-core will not flag input elements that don't have it. Axe-core only flags invalid values for autocomplete.

**Comment by asthaawasthi-laitkor** [comment]
Thank you @straker.
